spec/views/shared/_help-tooltip.html.erb_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+describe 'shared/_help-tooltip.html.erb' do
+  it 'renders with block as USWDS tooltip' do
+    tooltip_text = 'foobar'
```

Comment:
WDYT of adding some HTML-unsafe characters in this like quotes so that we can make sure it's properly escaped in the check below?

```suggestion
    tooltip_text = '"foo" "bar"'
```

Reply:
> WDYT of adding some HTML-unsafe characters in this like quotes so that we can make sure it's properly escaped in the check below?

Makes sense. Updated in b17c579.